job-server/src/main/scala/spark/jobserver/AkkaClusterSupervisorActor.scala

```diff
@@ -229,49 +129,43 @@ class AkkaClusterSupervisorActor(daoActor: ActorRef) extends InstrumentedActor {
     }
 
     if (processStart.isSuccess) {
-      contextInitInfos(contextActorName) = (isAdHoc, successFunc, failureFunc)
+      contextInitInfos(actorName) = (isAdHoc, successFunc, failureFunc)
     } else {
       failureFunc(processStart.failed.get)
     }
-
   }
 
-  private def createContextDir(name: String,
-                               contextConfig: Config,
-                               isAdHoc: Boolean,
-                               actorName: String): java.io.File = {
+  private def createContextDir(
```

Comment:
Personally I prefer the original style, where the first param is on the same line as the function name.